pom.xml

```diff
@@ -34,6 +34,8 @@
 		<!-- DB Driver Versions -->
 		<h2.driver.version>2.2.224</h2.driver.version>
 		<db2.driver.version>11.5.9.0</db2.driver.version>
+<!--		<oracle.driver.version>18.3.0.0</oracle.driver.version>-->
+<!--		<oracle.driver.jdkversion>8</oracle.driver.jdkversion>-->
```

Comment:
Dit is voor de backport handig maar niet meer nodig in de master toch?

Reply:
Oh die change is per ongeluk gecommit. Ik wou wat dingen testen tegen oude drivers, en snel kunnen switchen. Even over het hoofd gezien dat het per ongeluk toch gecommit is.